package.json

```diff
@@ -41,20 +45,22 @@
     "rimraf": "^2.6.3"
   },
   "devDependencies": {
+    "@babel/runtime": "^7.4.5",
```

Comment:
Guessing that Babel is required for jest? Something that we were previously getting 'for free' w/ `create-react-app`?

Reply:
Yes - instead of doing a straight tsc compile for jest we are using babel.  We are getting a better jest debugging experience doing this for some reason that I don't fully understand.